### PR TITLE
Quiz and rules page

### PR DESCRIPTION
**Pull request by abhishektanwar**

### CODE REVIEW

File: CSS/quiz.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.quiz-container{
+	align-items: center;
+}
+
+.question-container{
+	border-radius: 15px;
+	background-color: var(--dark-primary-color);
+	height: fit-content;
+	width: 600px;
+	margin-top: 30px;
```
Comment (Ranjeetk489) [inline CSS/quiz.css:14]: use rem

**Review by Ranjeetk489 — APPROVED** [review]
remove comments
use rem in place of px
after the changes code will be good to merge

File: CSS/quiz.css (hunk 1 above)
Comment (abhishektanwar) [reply CSS/quiz.css:14]: fixed in #3

File: CSS/quiz.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+body{
+	background-color: var(--light-primary-color);
+}
+
+.quiz-container{
+	align-items: center;
+}
+
+.question-container{
+	border-radius: 15px;
```
Comment (dev-ashishsethi) [inline CSS/quiz.css:10]: use rems

File: CSS/quiz.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	padding: 30px;
+}
+
+.options-container{
+	margin-top: auto;
+}
+
+.question-container button{
+	width: 100%;
+	margin-top: 20px;
```
Comment (dev-ashishsethi) [inline CSS/quiz.css:24]: use rem for length greater than 10px

File: CSS/quizes.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+}
+#prev1,#prev2{
+	position: absolute;
+	left: 30px;
+	top: 0;
+	height: 100%;
+    opacity: 0.7;
+    background: white;
+    color: black;
+}
```
Comment (dev-ashishsethi) [inline CSS/quizes.css:37]: don't style using id

File: CSS/quizes.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+.slider{
+	display: grid;
+	/* grid-template-columns: repeat(3,200px); */
```
Comment (dev-ashishsethi) [inline CSS/quizes.css:51]: avoid commenting code

File: CSS/rules.css [hunk 6]
```diff
@@ -0,0 +1,40 @@
+body{
+	background-color: var(--light-primary-color);
+}
+
+.rules-container{
+	height: calc(100vh - 10vh);
```
Comment (dev-ashishsethi) [inline CSS/rules.css:6]: could have used 90vh instead

File: CSS/rules.css [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+	justify-content: center;
+}
+
+.rules{
+	background-color: var(--dark-primary-color);
+	height: 600px;
+	width: 700px;
+	border-radius: 15px;
+	padding: 20px;
+	color: var(--white-color);
```
Comment (dev-ashishsethi) [inline CSS/rules.css:17]: use capitals for variable name

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

**Pull request merged**